escape_rooms/demo/calendar_event.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+  <record id="calendar_event_4" model="calendar.event">
+    <field name="name">John Smith - 🕵️ The Holmes Paradox Booking</field>
+    <field name="user_id" ref="base.user_admin"/>
+    <field name="location">MyCompany, Rue des Bourlottes 9, 1367 Ramillies, Belgium</field>
```

Comment:
```suggestion
   <field name="location" eval="ref('base.main_partner').contact_address"/>
```
Will this work? instead of giving static values. Here main_partner will be the partner of company.. you can find the exact XMLID.